js/TableDetailComponent.js

```diff
@@ -88,6 +118,15 @@ class TableDetailComponent {
 
         // Open panel
         this.$detailPanel.addClass('table-detail--open');
+
+        // Make elements focusable again
+        this.$detailPanel
+            .find('a[href], area[href], input, select, textarea, button, iframe, object, embed, [tabindex], *[contenteditable]')
```

Comment:
Duplicate lists of element types here, would be betters as a var/const

Reply:
Done